README.md

```diff
@@ -61,9 +61,13 @@
 - **Navigation menu**
```

Comment:
## To Improve 🚀 

I've noticed that the website is not fully responsive at some screen resolutions
![image](https://user-images.githubusercontent.com/68030297/223221398-73bbd574-76f5-4216-b195-e725803b47c1.png)

All sections suppose to fill the screen width with slight spaces on both the right and left sides of the section so I highly suggest working on those deflected sections since this website is your portfolio that will represent your professional.

Well done 👏🏼 and good luck 👍🏼